management/src/main/java/dk/kvalitetsit/itukt/management/boundary/mapping/dsl/ExpressionModelDslMapper.java

```diff
@@ -34,4 +34,57 @@ private String map(BinaryExpression.Operator operator) {
             case OR -> "eller";
         };
     }
+
+    private String map(ExistingDrugMedicationConditionExpression expression) {
+        return "EKSISTERENDE_LÆGEMIDDEL = {ATC = " + expression.atcCode() + ", FORM = " + expression.formCode() + ", ROUTE = " + expression.routeOfAdministrationCode() + "}";
+    }
+
+    private String map(BinaryExpression expression) {
+        if (isChain(expression, ExistingDrugMedicationConditionExpression.class)) {
+            List<ExistingDrugMedicationConditionExpression> collected = collect(expression, ExistingDrugMedicationConditionExpression.class);
+            String joined = collected.stream()
+                    .map(e -> String.format("{ATC = %s, FORM = %s, ROUTE = %s}",
+                            e.atcCode(), e.formCode(), e.routeOfAdministrationCode()))
+                    .collect(Collectors.joining(", "));
+            return "EKSISTERENDE_LÆGEMIDDEL i [" + joined + "]";
+        }
+
+        if (isChain(expression, IndicationConditionExpression.class)) {
+            List<IndicationConditionExpression> collected = collect(expression, IndicationConditionExpression.class);
+            String joined = collected.stream()
+                    .map(IndicationConditionExpression::requiredValue)
+                    .collect(Collectors.joining(", "));
+            return "INDICATION i [" + joined + "]";
+        }
+
+        return map(expression.left()) + " " + map(expression.operator()) + " " + map(expression.right());
+    }
+
+    private <T extends Expression> boolean isChain(Expression expr, Class<T> clazz) {
+        if (clazz.isInstance(expr)) return true;
+        if (expr instanceof BinaryExpression(
+                Expression left, BinaryExpression.Operator operator, Expression right
+        ) && operator == BinaryExpression.Operator.OR) {
```

Comment:
Synes formateringen gør det lidt svært at se hvor parenteserne hører til :)
```suggestion
        if (expr instanceof BinaryExpression(Expression left, BinaryExpression.Operator operator, Expression right)
                && operator == BinaryExpression.Operator.OR) {
```